server/service/packs_test.go

```diff
@@ -0,0 +1,30 @@
+package service
+
+import (
+	"context"
+	"testing"
+
+	"github.com/fleetdm/fleet/v4/server/datastore/mysql"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/fleetdm/fleet/v4/server/test"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetPack(t *testing.T) {
+	ds := mysql.CreateMySQLDS(t)
+	defer ds.Close()
```

Comment:
`ds.Close()` is now in the cleanup, so this is not needed.

That said, at this level, unless explicitly needed, the main goal is to test the service, so the ds is usually the mock one. I think that makes sense if we do the following:

- ds is tested against mysql directly, so that would cover `svc.ds.Pack(ctx, id)` above.
- at the service level we test the auth and that we use the ds correctly.
- then at the integration level or at the fleetctl level, we test that the endpoint works as expected (specially when there's some of the more "complex" decodes).

That's roughly the goal, let me know what you think.